### PR TITLE
Changelog for 1.14.10 and 1.15.6

### PR DESCRIPTION
**Pull request by mladlow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -321,6 +321,49 @@ leading to failure to complete merkle sync without a full re-index. [[GH-23013](
...
+* auth/cert: Cache trusted certs to reduce memory usage and improve performance of logins. [[GH-25421](https://github.com/hashicorp/vault/pull/25421)]
+* ui: Add `deletion_allowed` param to transformations and include `tokenization` as a type option [[GH-25436](https://github.com/hashicorp/vault/pull/25436)]
+* ui: redirect back to current route after reauthentication when token expires [[GH-25335](https://github.com/hashicorp/vault/pull/25335)]
+* ui: remove unnecessary OpenAPI calls for unmanaged auth methods [[GH-25364](https://github.com/hashicorp/vault/pull/25364)]
+
+BUG FIXES:
+
+* agent: Fix issue where Vault Agent was unable to render KVv2 secrets with delete_version_after set. [[GH-25387](https://github.com/hashicorp/vault/pull/25387)]
+* audit: Handle a potential panic while formatting audit entries for an audit log [[GH-25605](https://github.com/hashicorp/vault/pull/25605)]
+* core (enterprise): Fix a deadlock that can occur on performance secondary clusters when there are many mounts and a mount is deleted or filtered [[GH-25448](https://github.com/hashicorp/vault/pull/25448)]
```
Comment (anwittin) [inline CHANGELOG.md:355]: Do enterprise fixes usually have vault pr added to them?

Comment (anwittin) [inline CHANGELOG.md:355]: Is this an enterprise-only fix? It has a CE PR attached with docs updates that read as CE.

File: CHANGELOG.md [hunk 2]
```diff
@@ -757,6 +800,39 @@ sdk/ldaputil: use EscapeLDAPValue implementation from cap/ldap [[GH-22249](https
...
+
+IMPROVEMENTS:
+
+* auth/cert: Cache trusted certs to reduce memory usage and improve performance of logins. [[GH-25421](https://github.com/hashicorp/vault/pull/25421)]
+* ui: redirect back to current route after reauthentication when token expires [[GH-25335](https://github.com/hashicorp/vault/pull/25335)]
+* ui: remove unnecessary OpenAPI calls for unmanaged auth methods [[GH-25364](https://github.com/hashicorp/vault/pull/25364)]
+
+BUG FIXES:
+
+* core (enterprise): Fix a deadlock that can occur on performance secondary clusters when there are many mounts and a mount is deleted or filtered [[GH-25448](https://github.com/hashicorp/vault/pull/25448)]
```
Comment (anwittin) [inline CHANGELOG.md:830]: Same as the above comment.

**Review by anwittin — COMMENTED** [review]
There is a core change that may be incorrectly labeled

File: CHANGELOG.md (hunk 2 above)
Comment (mladlow) [reply CHANGELOG.md:830]: Going to resolve this one because it's the same entry

File: CHANGELOG.md (hunk 1 above)
Comment (mladlow) [reply CHANGELOG.md:355]: I think this is correct from the PR description. Performance replication is an enterprise feature, but it seems that most of the code changes were on the community edition side and there was just a test on the ENT side.

**Review by anwittin — APPROVED** [review]

**Pull request merged**